lib/rubygems/commands/precompile_command.rb

```diff
@@ -11,6 +11,14 @@ def initialize
     add_option('-a','--arch-dirs','Adds the architecture sub-folders to the output directory before writing') do |arch, options|
       options[:arch] = true
     end
+
+    add_option('-n','--noclean','do not clean up tempdirs, leaves temp dirs in place etc for inspection and debugging') do |noclean, options|
+      options[:debug] = true
+    end
+
+    add_option('-b','--build-config=CONFIG','commands to pass to the build i.e the run of extconf.rb') do |config, options|
+      options[:build_config] = config.split(' ')
```

Comment:
Would this be better suited to using `Shellwords.split` (for e.g. quoted strings?)